holidays/countries/israel.py

```diff
@@ -83,42 +101,42 @@ def _populate_optional_holidays(self):
         if self._year >= 2101:
             raise NotImplementedError
 
-        # Sukkot Chol HaMoed
-        sukkot_dt = self._get_holiday("SUKKOT")
+        sukkot_dt = self._get_holiday(SUKKOT)
         for offset in range(1, 6):
-            self._add_holiday("Sukkot", sukkot_dt + td(days=offset))
+            # Chol HaMoed Sukkot (Feast of Tabernacles holiday).
+            self._add_holiday(tr("חול המועד סוכות"), sukkot_dt + td(days=offset))
 
         if self._year >= 2008:
-            # Sigd
-            self._add_holiday("Sigd", self._get_holiday("YOM_KIPPUR") + td(days=+49))
+            # Sigd.
+            self._add_holiday(tr("סיגד"), self._get_holiday(YOM_KIPPUR) + td(days=+49))
 
-        # Purim
-        self._add_holiday("Purim", self._get_holiday("PURIM"))
+        # Purim.
+        self._add_holiday(tr("פורים"), self._get_holiday(PURIM))
 
-        # Passover (Chol HaMoed Pesach)
-        passover_dt = self._get_holiday("PASSOVER")
+        passover_dt = self._get_holiday(PASSOVER)
         for offset in range(1, 6):
-            self._add_holiday("Passover", passover_dt + td(days=offset))
+            # Chol HaMoed Pesach (Passover holiday).
+            self._add_holiday(tr("חול המועד פסח"), passover_dt + td(days=offset))
 
-        # Memorial Day
         if self._year >= 1963:
-            name = "Memorial Day"
-            memorial_day_dt = self._get_holiday("INDEPENDENCE_DAY") + td(days=-1)
+            # Yom Hazikaron (Fallen Soldiers and Victims of Terrorism Remembrance Day).
+            name = tr("יום הזיכרון לחללי מערכות ישראל ונפגעי פעולות האיבה")
```

Comment:
Generally this is just יום הזיכרון
but the rest is fine

Reply:
but that might be just how i refer to it! you are probably right that this is the official name!